### PR TITLE
added unicode emoji support for plant and pick

### PR DESCRIPTION
**Pull request by mikytron123**

### CODE REVIEW

File: NadekoBot.Core/Modules/Gambling/Services/PlantPickService.cs [hunk 1]
```diff
@@ -298,8 +402,8 @@ public async Task<long> PickAsync(ulong gid, ITextChannel ch, ulong uid, string
 
         public async Task<ulong?> SendPlantMessageAsync(ulong gid, IMessageChannel ch, string user, long amount, string pass)
         {
-            try
-            {
+           //try
+           //{
```
Comment (Gibstick) [inline NadekoBot.Core/Modules/Gambling/Services/PlantPickService.cs]: Why was this removed?

File: NadekoBot.Core/Modules/Gambling/Services/PlantPickService.cs [hunk 2]
```diff
@@ -321,20 +425,20 @@ public async Task<long> PickAsync(ulong gid, ITextChannel ch, ulong uid, string
...
         }
 
         public async Task<bool> PlantAsync(ulong gid, IMessageChannel ch, ulong uid, string user, long amount, string pass)
         {
             // normalize it - no more than 10 chars, uppercase
-            pass = pass?.Trim().TrimTo(10, hideDots: true).ToUpperInvariant();
+            pass = pass?.Trim().ToUpperInvariant();
             // has to be either null or alphanumeric
-            if (!string.IsNullOrWhiteSpace(pass) && !pass.IsAlphaNumeric())
+            if (string.IsNullOrWhiteSpace(pass))// && !pass.IsAlphaNumeric())
```
Comment (Gibstick) [inline NadekoBot.Core/Modules/Gambling/Services/PlantPickService.cs]: Remove commented code

**Review by Gibstick — COMMENTED** [review]
Is the license on the font OK?

**Comment by mikytron123** [comment]
> Is the license on the font OK?

Yes its from here https://github.com/eosrei/twemoji-color-font.
License here https://github.com/eosrei/twemoji-color-font/blob/master/LICENSE.md

File: NadekoBot.Core/Modules/Gambling/Services/PlantPickService.cs (hunk 1 above)
Comment (mikytron123) [reply NadekoBot.Core/Modules/Gambling/Services/PlantPickService.cs]: Fixed

File: NadekoBot.Core/Modules/Gambling/Services/PlantPickService.cs (hunk 2 above)
Comment (mikytron123) [reply NadekoBot.Core/Modules/Gambling/Services/PlantPickService.cs]: Fixed

**Review by Gibstick — APPROVED** [review]
